src/components/CalendarDay.jsx

```diff
@@ -58,7 +58,11 @@ class CalendarDay extends React.PureComponent {
     const { isFocused, tabIndex } = this.props;
     if (tabIndex === 0) {
       if (isFocused || tabIndex !== prevProps.tabIndex) {
-        this.buttonRef.focus();
+        requestAnimationFrame(() => {
```

Comment:
should we `cancelAnimationFrame` when the component unmounts?

Reply:
I don't think we need to, since I included a check that this.buttonRef is still truthy, so if it unmounts in between, it will essentially be a noop anyway.